app/javascript/angular/code/services/google/_map.js

```diff
@@ -149,15 +151,37 @@ export const map = (
       return newMarker;
     },
 
-    drawCustomMarker: function({ latLng, content, colorClass, callback, type }) {
-      const customMarker = buildCustomMarker(latLng, content, colorClass, callback, type);
+    drawCustomMarker: function({ latLng, content, colorClass, callback, type, objectId, value }) {
+      const customMarker = buildCustomMarker(latLng, content, colorClass, callback, type, objectId, value );
```

Comment:
Have you thought about passing the entire obj instead of destructuring and passing each as a single arg?